tests/nightly/gpu/test_anti_scaling.py

```diff
@@ -0,0 +1,180 @@
+                self.BART_OPT,
+                self.WIDE_DISTILLATION_OPT,
+                'DistillBartAgent',
+                False,
+                {
+                    'dec_hid_loss': 2.731,
+                    'enc_hid_loss': 0.0383,
+                    'enc_loss': 0.0383,
+                    'loss': 32.16,
+                    'pred_loss': 29.35,
+                },
+            ),
+            (
+                self.TRANSFORMER_OPT,
+                self.NARROW_DISTILLATION_DUMMY_LOSS_OPT,
+                'DistillNarrowTransformerAgent',
+                True,
+                {
+                    'dec_emb_loss': 1.625,
+                    'dec_hid_loss': 49.08,
+                    'dec_self_attn_loss': 3.609,
+                    'enc_dec_attn_loss': 29.65,
+                    'enc_emb_loss': 1.027,
+                    'enc_hid_loss': 0.5528,
+                    'enc_loss': 0.5478,
+                    'enc_self_attn_loss': np.inf,
+                    'loss': 11.41,
+                    'pred_loss': 9.238,
+                },
+            ),
+            (
+                self.BART_OPT,
+                self.NARROW_DISTILLATION_DUMMY_LOSS_OPT,
+                'DistillNarrowBartAgent',
+                True,
+                {
+                    'dec_emb_loss': 9.495,
+                    'dec_hid_loss': 0.6304,
+                    'dec_self_attn_loss': 514.2,
+                    'enc_dec_attn_loss': 137.6,
+                    'enc_emb_loss': 6.642,
+                    'enc_hid_loss': 0.05015,
+                    'enc_loss': 0.05089,
+                    'enc_self_attn_loss': 9.318,
+                    'loss': 11.39,
+                    'pred_loss': 8.918,
+                },
+            ),
+        ]
+        for (
+            model_opt,
+            distillation_opt,
+            model_name,
+            is_tinybert_style,
+            desired_losses,
+        ) in opts_and_desired_losses:
+            opt = {
+                **self.BASE_OPT,
+                **model_opt,
+                **distillation_opt,
+                'model': f'{self.DISTILLATION_MODEL_PREFIX}:{model_name}',
+                'num_examples': 1,
+                'skip_generation': True,
+                'hidden_loss_coeff': 1,
+                'encoder_loss_coeff': 1,
+                'pred_loss_coeff': 1,
+                'task_loss_coeff': 1,
+            }
+            valid, _ = testing_utils.eval_model(Opt(opt), skip_test=True)
+            if not is_tinybert_style or precise_mode:
+                for loss_name, desired_loss in desired_losses.items():
+                    if np.isinf(desired_loss):
+                        self.assertTrue(np.isinf(valid[loss_name].value()))
+                    else:
+                        if abs(valid[loss_name].value() / desired_loss - 1) > 0.01:
+                            raise ValueError(
+                                f"""\
+Error in matching {loss_name} for {model_name}! \
```

Comment:
Use multiline strings please, not \

Reply:
```python
ValueError(
    f'notice the lack of commas '
    f'or \\ for multiline strings'
)
```

Reply:
Whoops, yes, removing the 2 extra backslashes